packages/apollo-server-azure-functions/src/ApolloServer.ts

```diff
@@ -25,12 +25,6 @@ export class ApolloServer extends ApolloServerBase {
   // another place, since the documentation becomes much more complicated when
   // the constructor is not longer shared between all integration
   constructor(options: Config) {
-    if (process.env.ENGINE_API_KEY || options.engine) {
-      options.engine = {
-        sendReportsImmediately: true,
```

Comment:
It was removed because it's not necessary with shutdown being handled gracefully by the agent, and it can impact performance of the server (and our infrastructure). 

I'll take it out of this PR, though.